src/CADL.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -359,6 +398,16 @@ function loadOperation(
     }
 }
 
+function isLROOperation(op: OperationDetails) {
+    return false;
+}
+function isPagingOperation(op: OperationDetails) {
+    return false;
+}
+function isSupportedOperation(op: OperationDetails) {
+    if (isLROOperation(op) || isPagingOperation(op)) return false;
```

Comment:
`return !isLROOperation(op) && !isPagingOperation(op)`?

Reply:
Thanks. but We will add other supported scenarios. so we will not directly return at first statement.